Casks/h/horos.rb

```diff
@@ -1,22 +1,44 @@
 cask "horos" do
-  on_el_capitan :or_older do
-    version "2.0.2"
-    sha256 "5cc1d6c71c8ae643b4df4fecee93dbe3cfacbcffef52001a76a7683a2725ac08"
+  arch arm: "arm64"
```

Comment:
Why not this and keep the URL the same?
```suggestion
  arch arm: "_arm64"
```

Reply:
This has been changed in a previous review: https://github.com/Homebrew/homebrew-cask/pull/177520#discussion_r1650068305.

However, keeping the download URL the same does not seem possible since the arm version has an extra underscore before the version information.